src/objects/api/v2/filters.py

```diff
@@ -124,7 +124,9 @@ def filter_data_icontains(self, queryset, name, value: str):
         return queryset.extra(where=[where_str], params=[value])
 
     def filter_date(self, queryset, name, value: date_):
-        return queryset.filter_for_date(value)
+        """actual filtering is performed in the viewset"""
+        return queryset
 
     def filter_registration_date(self, queryset, name, value: date_):
-        return queryset.filter_for_registration_date(value)
+        """actual filtering is performed in the viewset"""
```

Comment:
Same as question as filter_date

Reply:
Same thing - It's added to the filterset just to generate the OAS with correct query params